src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1183,10 +981,13 @@ describe('useUserSubsidyApplicableToCourse', () => {
     });
   });
 
-  it.each([
-    { enterpriseAdminUsers: [] },
-    { enterpriseAdminUsers: ['edx@example.com'] },
-  ])('does not have redeemable subsidy access policy and catalog(s) contains course (%s)', async ({ enterpriseAdminUsers }) => {
+  // TODO Fix test, shows 'No matching tests found'
```

Comment:
Is this still a valid TODO? As is, your changes make this a single test case due vs. the previous 2 due to the singular array element.

Reply:
Removed 👍🏽